builder/Makefile.tracee

```diff
@@ -40,6 +40,7 @@ help:
 	@echo "To execute tracee container(s):"
 	@echo ""
 	@echo "    $$ make -f builder/Makefile.tracee run-alpine-tracee-core"
+	@echo "    $$ make -f builder/Makefile.tracee run-alpine-tracee-core-btfhub"
 	@echo "    $$ make -f builder/Makefile.tracee run-alpine-tracee-nocore"
 	@echo ""
```

Comment:
Okay, @danielpacak and @grantseltzer ... as we spoke today, those are the 3 container images we should have IMO. Check this git commit description.